docs/index.md

```diff
@@ -1,11 +1,6 @@
 (SQLair documentation root file)=
 # SQLair
 
-```{note}
-The project is still under development and there may be still be major changes
-to the API.
```

Comment:
I see you've chosen to get rid of this entirely.

Reply:
Yea, I think not quite important enough to go at the top of the page, and as you mentioned before, I've changed the wording a bit.